docs/jenkins.md

```diff
@@ -54,25 +54,14 @@ https://github.com/theforeman/foreman-infra/tree/master/puppet/modules contains
 
 ### Configuring a new node
 
-For Enterprise Linux 8:
+For Enterprise Linux 9, ensure the correct Puppet repository is configured:
```

Comment:
I think I'd drop the version and the "ensure" as that's already in the first step